modules/monitoring-searching-alerts-silences-and-alerting-rules.adoc

```diff
@@ -0,0 +1,72 @@
+** *Info*. The alert is provided for informational purposes only.
+** *None*. The alert has no defined severity.
+** You can also create custom severity definitions for alerts relating to user-defined projects.
+
+* *Source* filters:
+** *Platform*. Platform-level alerts relate only to default {product-title} projects. These projects provide core {platform-title} functionality.
+** *User*. User alerts relate to user-defined projects. These alerts are user-created and are customizable. User-defined workload monitoring can be enabled post-installation to provide observability into your own workloads.
+
+[discrete]
+== Understanding silence filters
+
+In the Administrator perspective, the *Silences* page in the Alerting UI provides details about silences applied to alerts in default {product-title} and user-defined projects. The page includes a summary of each silence's state and the time at which a silence ends.
+
+You can filter by silence state. By default, only *Active* and *Pending* silences are displayed. The following describes each silence state filter option:
+
+* *Silence State* filters:
+** *Active*. The silence is active and the alert will be muted until the silence is expired.
+** *Pending*. The silence has been scheduled and it is not yet active.
+** *Expired*. The silence has expired and notifications will be sent if the alert's conditions are true.
+
+[discrete]
+== Understanding alerting rule filters
+
+In the Administrator perspective, the *Alerting Rules* page in the Alerting UI provides details about alerting rules relating to default {product-title} and user-defined projects. The page includes a summary of each alerting rule's state, severity and source.
+
+You can filter by rule state, severity and source. By default, only *Platform* alerting rules are displayed. The following describes each alerting rule filtering option:
+
+* *Rule State* filters:
+** *Active*. The alerting rule is active and alerts are fired when its conditions are true.
+** *Inactive*. The alerting rule is inactive.
+
+* *Severity* filters:
+** *Critical*. The conditions defined in the alerting rule could have a critical impact. When true, these conditions require immediate attention. Alerts relating to the rule are typically paged to an individual or to a critical response team.
+** *Warning*. The conditions defined in the alerting rule might require attention in order to prevent a problem from occurring. Alerts relating to the rule are typically routed to a ticketing system for non-immediate review.
+** *Info*. The alerting rule provides informational alerts only.
+** *None*. The alerting rule has no defined severity.
+** You can also create custom severity definitions for alerting rules relating to user-defined projects.
+
+* *Source* filters:
+** *Platform*. Platform-level alerting rules relate only to default {product-title} projects. These projects provide core {platform-title} functionality.
```

Comment:
Same here re: `{platform-title}` showing up verbatim.

Reply:
Resolved throughout the PR.